sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -250,14 +250,13 @@ def _generate_select_expression(
                             # unnest repeated record
                             select_expr.append(
                                 f"""
-                                    (
-                                        SELECT ARRAY_AGG(
+                                    ARRAY(
+                                        SELECT
```

Comment:
This change wasn't strictly necessary, but I figured while I'm modifying this code I'd take the opportunity to make this slight improvement.